src/migrations.js

```diff
@@ -54,12 +54,23 @@ exports.Migrations= {
     for(var domain in action_map.getItemClones()){
       if(badger.storage.getNextUpdateForDomain(domain) === 0){
         // Recheck at a random time in the next week
-        var recheckTime = utils.getRandom(Date.now(), Date.now() + (utils.oneDay() * 7));
+        var recheckTime = utils.getRandom(utils.nDaysFromNow(7), utils.nDaysFromNow(14));
```

Comment:
The original logic seems fine (sometime between now and seven days from now) ... We're using the new wrapper functions but with a changed range.